R/sets.r

```diff
@@ -1,11 +1,16 @@
 #' Set operations
 #'
-#' These functions override the set functions provided in base to make them
-#' generic so that efficient versions for data frames and other tables can be
-#' provided. The default methods call the base versions. Beware that
-#' `intersect()`, `union()` and `setdiff()` remove duplicates.
+#' @description
+#' Perform set operations using the rows of a data frame or tibble.
```

Comment:
```suggestion
#' Perform set operations using the rows of a data frame.
```

Feels like that goes without saying